libs/shared/ui/src/lib/components/inputs/input-text/input-text.tsx

```diff
@@ -74,7 +76,7 @@ export function InputText(props: InputTextProps) {
               id={label}
               className="input__value"
               value={currentValue}
-              type={type}
+              type={currentType}
```

Comment:
I don't get why this change here?

Reply:
Because I need an local "type" when I changed between "password" and "type"